docs/vscode-extension/envchecker-help.md

```diff
@@ -49,15 +51,29 @@ Please refer to [the official website](https://docs.microsoft.com/en-us/azure/az
 As the Teams Toolkit project is implemented by `Node.js`, it's required to install the npm pacakges and run the project in local. 
 
 ### Mitigation
-Please refer to [nodejs.org](https://nodejs.org/) to install the right version: currently only LTS versions (v10, v12 and v14) are supported by Teams Toolkit, and `Node v14` would be recommended to be installed.
+Please refer to [nodejs.org](https://nodejs.org/) to install the right version: currently only LTS versions (v10, v12, v14 and v16) are supported by Teams Toolkit that does not containing Azure Functions or SPFx, and `Node v14` would be recommended to be installed.
```

Comment:
1. v10 is not LTS currently.
2. Consider using https://nodejs.org/en/about/releases/ instead of https://nodejs.org/, see [this bug](https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/12684771).